citi-onboarding/MeuApe-onepage
# feature/benefitBack: structuring the back of the benefits section

## What I did:

- Basically I structured the back of the benefits sections for customers and builders
- I performed the tests with the database integration

## How to test
1. Open the terminal and go to the folder of this project

~~~~typescript
git checkout develop
~~~~
~~~~typescript
git pull
~~~~
~~~~typescript
git checkout feature/benefitBack
~~~~
~~~~typescript
git pull origin feature/benefitBack
~~~~
~~~~typescript
yarn install or npm install
~~~~
~~~~typescript
yarn start or npm start
~~~~
~~~~typescript
docker compose up
~~~~

2. In your browser go to <http://localhost:3000>

3. Check if the component looks just like the mockup in different screen sizes

4. Read the changed files section on GitHub

5. Test the things that you think are worth testing, even the ones that are not in this description
